src/cargo/sources/registry/index/cache.rs

```diff
@@ -212,3 +218,48 @@ impl<'a> SummariesCache<'a> {
         contents
     }
 }
+
+/// Manages the on-disk index caches.
+pub struct CacheManager<'gctx> {
+    /// [`GlobalContext`] reference for convenience.
+    pub gctx: &'gctx GlobalContext,
```

Comment:
Feels weird to me that callers access the context through this

Reply:
Fixed!

Reply:
Passing in individual unstable features seems a bit higher churn as we evolve but whichever way.